explainers/periodicsync-explainer.md

```diff
@@ -0,0 +1,178 @@
+# Periodic Background Sync
+
+**Written**: 2019-03-26<br/>
+**Updated**: 2019-04-17
+
+Periodic Background Sync is a method that enables web applications to periodically synchronize data
+in the background, building on the [Background Sync](https://wicg.github.io/BackgroundSync/spec/)
+specification that defines a method for one-off synchronization.
+
+The [original explainer](https://github.com/WICG/BackgroundSync/blob/master/explainer.md#periodic-synchronization-in-design)
+included a thorough exploration of this capability, but it neither got specified, nor ever shipped
+in a browsing engine. This was caused by a combination of low interest from both developers and
+other browser engines, and high complexity due to an unclear permission model. We believe the
+situation has changed since.
+
+If this proposal stands, we plan to extend the [Background Sync](https://wicg.github.io/BackgroundSync/spec/)
+specification to describe this ability as well, and move the document to the standardization track.
+
+## Use cases
+  Consider a website that's offline enabled. It uses service workers to provide an almost
```

Comment:
Why is this section indented? Only list items have to be indented.

Reply:
fixed.